src/Application.php

```diff
@@ -9,7 +9,6 @@
 
 namespace Zend\Expressive;
 
-use BadMethodCallException;
 use DomainException;
```

Comment:
If you're going to use the local `BadMethodCallException`, we should do so for `DomainException` as well. I'll take care of that on merge.